index.html

```diff
@@ -0,0 +1,97 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>CV - Pinchuk Vladislav</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style type="text/css">
+       div {
+        max-width: 1024px;
+        text-align: center;
+       } 
+    </style>
+</head>
+<body>
+    <div>
+    <header>
+        <h1>Pinchuk Vladislav Alexandrovich</h1>
+    </header>
+    <hr>
+    <img src="photo.jpg" width="150" alt="Photo Pinchuk Vladislav">
+     
+    <section>
+        <h3>Event Specialists / Event Manager <br>
+        <h4>Work experience - 11 years 8 months</h4>
+        </h3>
+        <p>Male, 30, born May 24, 1990</p>
+        <p><a href="tel:+375 (44) 515 49 83">+375 (44) 515 49 83</a><br>
```

Comment:
Hello, @Peshahod 

Looks like you use invalid tel format in href attr. Please check the [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/A)